### DISCUSSION TITLE
Need to visualise data in all user groups

### DISCUSSION THREAD

**Discussion by Mohanapriya777** [Sunbird-cQube/community#842] [General]
Hi,
We need to drill down the NVSK data till School level. For now it is only visualizing state officer level. What steps are required to display the data to users at every level(District officer, Block officer, etc..). Are there alternative schemas that need to be adopted for data ingestion?

**Comment by RNarmatha** [comment]
Hi @Mohanapriya777 

Starter pack programs can be visualized at the state level. State Program can be drill down at the school level
